### ISSUE TITLE
Import fails hard for skipped rows

### ISSUE THREAD

**Issue by tomsaen** [django-import-export/django-import-export#1937]
**Describe the bug**
Hey folks!

I am currently migrating towards 4.1.1 from 3.3.9 and I have encountered one problem I believe is a bug. When importing rows that are skipped due to e.g. no changes, the import process fails hard with a `KeyError: 'skip'` message. From what I can tell, this is only happening when running Django 5.1.0, because in that case, the [_log_actions](https://github.com/django-import-export/django-import-export/blob/main/import_export/admin.py#L226) method is being called. This one then calls the [_create_log_entries](https://github.com/django-import-export/django-import-export/blob/main/import_export/admin.py#L577-L585) method, which isn't really taking care of skipped rows, leading to the beforementioned `KeyError`.
In case of older Django versions, skipped rows are [correctly managed](https://github.com/django-import-export/django-import-export/blob/main/import_export/admin.py#L235-L238)

Full tack trace:
```python
Internal Server Error: /admin/revenues/invoices/process_import/
Traceback (most recent call last):
  File "/Users/tom/Code/backend/.venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/tom/Code/backend/.venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/tom/Code/backend/.venv/lib/python3.12/site-packages/django/utils/decorators.py", line 188, in _view_wrapper
    result = _process_exception(request, e)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/tom/Code/backend/.venv/lib/python3.12/site-packages/django/utils/decorators.py", line 186, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/tom/Code/backend/.venv/lib/python3.12/site-packages/django/views/decorators/cache.py", line 80, in _view_wrapper
    response = view_func(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/tom/Code/backend/.venv/lib/python3.12/site-packages/django/contrib/admin/sites.py", line 241, in inner
    return view(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/tom/Code/backend/selina/common/admin/import_tools.py", line 416, in process_import
    res = super().process_import(request, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/tom/Code/backend/.venv/lib/python3.12/site-packages/django/utils/decorators.py", line 48, in _wrapper
    return bound_method(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/tom/Code/backend/.venv/lib/python3.12/site-packages/django/views/decorators/http.py", line 64, in inner
    return func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/tom/Code/backend/.venv/lib/python3.12/site-packages/import_export/admin.py", line 178, in process_import
    return self.process_result(result, request)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/tom/Code/backend/.venv/lib/python3.12/site-packages/import_export/admin.py", line 201, in process_result
    self.generate_log_entries(result, request)
  File "/Users/tom/Code/backend/.venv/lib/python3.12/site-packages/import_export/admin.py", line 215, in generate_log_entries
    self._log_actions(result, request)
  File "/Users/tom/Code/backend/.venv/lib/python3.12/site-packages/import_export/admin.py", line 564, in _log_actions
    self._create_log_entries(request.user.pk, rows)
  File "/Users/tom/Code/backend/.venv/lib/python3.12/site-packages/import_export/admin.py", line 573, in _create_log_entries
    action_flag = logentry_map[import_type]
                  ~~~~~~~~~~~~^^^^^^^^^^^^^
KeyError: 'skip'
{"event": "Internal Server Error: /admin/invoices/process_import/", "exc_info": ["<class 'KeyError'>", "KeyError('skip')", "<traceback object at 0x17c1aad00>"]}
```

**To Reproduce**
1. Go to one resource that is exportable
2. Click on export
3. Click on import using the currently exported file
4. Confirm the skipped rows in the screen
5. Proceed with the import and watch if fail.

**Versions (please complete the following information):**
 - Django Import Export: 4.1.1
 - Python: 3.12.1
 - Django: 5.1

**Expected behavior**
Skip the rows marked as "skipped" and don't write any change log entries for them. 

**Screenshots**
-

**Additional context**
Please let me know if you need any more context!

**Comment by harishthampy** [comment]
@tomsaen Could you tell me more about this problem

**Comment by tomsaen** [comment]
Sure! What other information can I provide?

**Comment by matthewhegarty** [comment]
Thanks for raising - I have a fix for this in #1944
